index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+<div class="container">
+
+    <header class="header">
+        <h1 id="title" class="text-center">Website Builders America Survey Form</h1>
+        <p id="description" class="description text-center">
+            Thank you for taking the time to help us inprove our service!
+        </p>
+    </header>
+
+    <form id="survey-form">
+
+        <div class="form-group">
+            <label id="name-label" for="name">Name</label>
+            <input type="text" name="name" id="name" class="form-control" placeholder="Enter your name" required />
+        </div>
+
+        <div class="form-group">
+            <label id="email-label" for="email">Email</label>
+            <input type="email" name="email" id="email" class="form-control" placeholder="Enter your email" required />
+        </div>
+
+        <div class="form-group">
+            <label id="number-label" for="number">Age<span class="clue">(optional)</span></label>
+            <input type="number" name="age" id="number" min="0" max="100" class="form-control" placeholder="Age" />
+        </div>
+
+
```

Comment:
It is advisable to remove empty spaces

Reply:
- [ ] Great job, identifying that there is excess space issue here. It would be nice to explain why removing the extra space is recommended. Cheers 👍